MetadataExtractor/IO/BufferReader.Sequential.cs

```diff
@@ -114,17 +114,19 @@ public ulong GetUInt64()
 
     public string GetString(int bytesRequested, Encoding encoding)
     {
-        // This check is important on .NET Framework
+        if (bytesRequested < 0)
+            throw new ArgumentOutOfRangeException(nameof(bytesRequested), "Must be 0 or greater");
+
         if (bytesRequested is 0)
             return "";
 
-        Span<byte> bytes = bytesRequested <= 256
-            ? stackalloc byte[bytesRequested]
-            : new byte[bytesRequested];
+        using var buffer = bytesRequested <= 256
```

Comment:
What do you think about moving the 256 to a constant, like `ScopedBuffer.MaxStackBufferSize`?